arcaflow_plugin_stressng/stressng_schema.py

```diff
@@ -217,13 +233,17 @@ class CpuStressorParams(CommonStressorParams):
     ] = CpuMethod.ALL
 
     def to_jobfile(self) -> str:
+        params = schema.build_object_schema(CpuStressorParams).serialize(
+            CpuStressorParams(
+                Stressors(self.stressor),
+                self.workers,
+                self.cpu_ops,
+                self.cpu_load,
+                CpuMethod(self.cpu_method),
+            )
+        )
         result = f"cpu {self.workers}\n"
-        if self.cpu_ops is not None:
-            result += f"cpu-ops {self.cpu_ops}\n"
-        if self.cpu_load is not None:
-            result += f"cpu-load {self.cpu_load}\n"
-        if self.cpu_method is not None:
-            result += f"cpu-method {CpuMethod(self.cpu_method)}\n"
+        result += params_to_jobfile(params)
```

Comment:
This is cleaner and probably more clear in the code, but what I really wanted was to iterate through the parameters in the class and use the `schema.id` annotations as the keys in the dict, since we already go to the trouble of defining those (the importance here is the conversion of the `_` characters that the schema requires to the `-` characters that the stress-ng input file expects). I never came up with a good way to do this.